src/diffusers/loaders.py

```diff
@@ -1141,6 +1235,20 @@ def _modify_text_encoder(cls, text_encoder, lora_scale=1, network_alpha=None, ra
             )
             lora_parameters.extend(attn_module.out_proj.lora_linear_layer.parameters())
 
+        if patch_aux:
+            for _, aux_module in text_encoder_aux_modules(text_encoder):
+                aux_module.fc1 = PatchedLoraProjection(
+                    aux_module.fc1, lora_scale, network_alpha, rank=rank, dtype=dtype
+                )
+                lora_parameters.extend(aux_module.fc1.lora_linear_layer.parameters())
+
+                aux_module.fc2 = PatchedLoraProjection(
+                    aux_module.fc2, lora_scale, network_alpha, rank=rank, dtype=dtype
+                )
+                lora_parameters.extend(aux_module.fc2.lora_linear_layer.parameters())
+
+            text_encoder.aux_state_dict_populated = True
```

Comment:
Setting auxiliary flags on objects like this is a bit of an anti-pattern. If there is another part of the PR that requires knowing if we are patching more than just the attention linear layers, how can we be more explicit about letting that part of the code know?